superset/views/core.py

```diff
@@ -2565,7 +2565,8 @@ def sql_json(self):
                 data = sql_lab.get_sql_results(
                     query_id,
                     rendered_query,
-                    return_results=True)
+                    return_results=True,
+                    user_name=g.user.username)
```

Comment:
To be sure, let's `user_name=g.user.username if g.user else None` as some environments may allow anonymous access

Reply:
Otherwise LGTM

Reply:
got it.